app/forms/placement_form.rb

```diff
@@ -38,7 +44,33 @@ def title
   end
 
   def save!
```

Comment:
I think this form will need to support stashing. If the user decides to go back to earlier steps to change the information, we'll need to hydrate the values into the form. Saving should probably be done in the end on the confirm page once the user confirms, until then we'll need to stash/undo stuff if they go back and forth to change information.

The base `TraineeForm` usually provides that out of the box but the DegreesForm has a different implementation of it which may be relevant for placements. We could cover all of this it in a sep. ticket? (is there a sep ticket?)

Reply:
Ah yes, at first glance I thought the prototype was saving in one step but there's an option to cancel so stashing is needed as you say. It looks like `DegreeForm` and `DegreesForm` work the way they do because the former represents an individual degree and the later the trainee (with it's collection of degrees). It makes sense to follow that pattern I think.

I could create a `PlacementsForm` to work with the `PlacementForm` that's already here but I've got a feeling that might be treading on some of the work you are doing on the other tickets. It might be best to log another ticket.